tests/unit/CoreQuoteTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace WPGraphQL\ContentBlocks\Unit;
+
+final class CoreQuoteTest extends PluginTestCase {
+	/**
+	 * The ID of the post created for the test.
+	 *
+	 * @var int
+	 */
+	public $post_id;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->post_id = wp_insert_post(
+			[
+				'post_title'   => 'Post with Quote',
+				'post_content' => '',
+				'post_status'  => 'publish',
+			]
+		);
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function tearDown(): void {
+		parent::tearDown();
+
+		wp_delete_post( $this->post_id, true );
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function query(): string {
+		return '
+			fragment CoreQuoteBlockFragment on CoreQuote {
+				attributes {
+					anchor
+					backgroundColor
+					citation
+					className
+					cssClassName
+					fontFamily
+					fontSize
+					gradient
+					# layout
+					lock
+					# metadata
+					style
+					# textAlign
+					textColor
+					value
+				}
+			}
+
+			query Post( $id: ID! ) {
+				post(id: $id, idType: DATABASE_ID) {
+					databaseId
+					editorBlocks {
+						apiVersion
+						blockEditorCategoryName
+						clientId
+						cssClassNames
+						innerBlocks {
+							name
+						}
+						isDynamic
+						name
+						parentClientId
+						renderedHtml
+						...CoreQuoteBlockFragment
+					}
+				}
+			}
+		';
+	}
+
+	/**
+	 * Test case for retrieving core quote block fields and attributes.
+	 *
+	 * The following aspects are tested:
+	 * - The absence of errors in the GraphQL response.
+	 * - Presence of the 'data' and 'post' keys in the response.
+	 * - Matching post ID.
+	 * - Correct block name ('core/quote').
+	 * - Correct retrieval of the block's attributes, especially 'citation', 'className', 'cssClassName' and 'value'.
+	 *
+	 * @return void
+	 */
+	public function test_retrieve_core_quote_fields_and_attributes() {
```

Comment:
See feedback on https://github.com/rtCamp/wp-graphql-content-blocks/pull/24 and https://github.com/rtCamp/wp-graphql-content-blocks/pull/22

Reply:
Fixed ✅